robots/flakefinder/report.go

```diff
@@ -355,13 +355,10 @@ func Report(results []*Result, reportOutputWriter *storage.Writer) error {
 // duplicates, thus if a test has data with several severities the highest one is picked, leading to an earlier
 // encounter in the slice.
 func SortTestsByRelevance(data map[string]map[string]*Details, tests []string) (testsSortedByRelevance []string) {
-	foundTests := map[string]bool{}
-	for _, test := range tests {
-		foundTests[test] = false
-	}
+	foundTests := map[string]struct{}{}
```

Comment:
This is fine as well, but you could have left the map as a map of bools.

The only thing was, the initialization loop was not needed. 

If you do `found = foundTests[test]` and the record is not there found is set to the zero value of the record, which is false. The second (optional) returned parameter is used to distinguish between a miss and a record containing false (which makes no difference here).

Reply:
Yeah, i could have. I just wanted to align the code to the pattern used [elsewhere](https://github.com/kubevirt/project-infra/blob/master/robots/flakefinder/report.go#L239). BTW I found using the default value when checking for presence of a value a little awkward.

Reply:
Yes I agree. Just wanted to be sure my comment was not mis-intepreted :-)

Reply:
Thanks for the explanation. I am indeed very grateful for any explanatory comment as I still am trying to find my way through the language :-)